backend/docs/development_docs/api_requests_per_screen.rst

```diff
@@ -0,0 +1,272 @@
+  four-tuples, *(name, UUID, association type, job function)*.
+* ``managers`` correspond to ``manager`` in MO and are triplets,
+  *(manager type, name, UUID)*.
+
+
+**NOTE:** The UUIs for employees, associated and managers are the person
+UUIDs, found under the MO API's ``/e/`` section.
+
+Employees
+.........
+
+The "wrapping" with response headers etc. is like for departments. A
+typical "doc" for a person could be: ::
+
+    {
+        "uuid":["cee8800a-983d-41fa-998c-b4557d68ec35"],
+        "name":["Anna Bjerre Reidl"],
+        "locations":["PHONE",
+          "21557342",
+          "DAR",
+          "Strandgårdsvej 16, Skallerup Klit, 9800 Hjørring",
+          "EMAIL",
+          "annar@hjorring.dk"],
+        "departments":["IT-Support",
+          "d3a9e589-5be0-4d28-95af-5d24ac42a2e9",
+          "Ansat",
+          "Specialist"],
+        "managing":["Direktør",
+          "IT-Support",
+          "d3a9e589-5be0-4d28-95af-5d24ac42a2e9"],
+        "id":"e6207b7c-6204-44e2-8839-277694589883",
+        "_version_":1616933478548373504
+    }
+
+
+This record has the following non-trivial members:
+
+* ``locations`` - this works as for departments.
+* ``departments`` - corresponds to ``engagement`` in MO. These are
+  to be interpreted as four-tuples consisting of *(department name,
+  UUID, engagement type, job function)*. Departments in which the person
+  is *employed*.
+* ``associated`` - corresponds to ``association`` in MO and organized as
+  ``departments``. Departments to which the person is associated.
+  Association type might e.g. be "Konsulent".
+  (tilknyttet).
+* ``managing`` - the departments which the user is managing. These are
+  triplets consisting of *(department name, UUID, manager type)*.
+  Manager type might e.g. be "Afdelingsleder" or "Direktør".
+
+
+Subdepartments/sections
+-----------------------
+
+The names and UUIDs of the departments/sections immediately under the
+root may be found in the root doc's ``department`` member, as described
+above. 
+    
+You can access the full data (including phone numbers, etc.) for each of
+these subdepartments by doing a query for the corresponding UUID, as we
+did with the root department.
+
+
+Display results (general query)
+===============================
+
+There are two separate URLs for searching departments, basically ::
+
+    http://127.0.0.1:8983/solr/employees/
+
+    http://127.0.0.1:8983/solr/departments/
+
+To search for *persons* or *employees* in all supported fields, write
+(e.g., searching for the name "Eva"): ::
+
+    http://127.0.0.1:8983/solr/employees/select?q=name:Eva%20locations:Eva%20departments:Eva%20associated:Eva%20managing:Eva
+
+I.e., this is an explicit search for the string in all of the fields
+that are supported for employees. 
+
+In order to search for *departments* in all fields, we do the
+corresponding search on the departments URL. 
+
+Since all employees (persons, Users in LoRa) are indexed under the employees URL
```

Comment:
You should include the explanation earlier, the first time you use the term.

```suggestion
Since all employees (persons, Users in LoRa) are indexed under the employees URL
```